docs/source/certs_management.md

```diff
@@ -0,0 +1,382 @@
+     │   ├── admincerts
+     │   │   └── cert.pem
+     │   ├── cacerts
+     │   │   └── localhost-7053.pem
+     │   ├── keystore
+     │   │   └── key.pem
+     │   ├── signcerts
+     │   │   └── cert.pem
+     │   |── user
+     └── tls
+</pre>
+
+**Impact if expired**: Production outage. Orderers do not start without a valid Enrollment Certificate.
+
+[Reference](https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#enroll-orderer)
+
+
+#### Orderer TLS Certificate
+
+**Description**: TLS Certificate for the ordering node communication.
+
+**Location**: Dependent on implementation:
+
+<pre>
+ordererca/
+└── orderer1
+    ├── msp
+    └── tls
+        ├── cacerts
+        ├── keystore
+        |   └── key.pem
+        ├── signcerts
+        │   └── cert.pem
+        ├── tlscacerts
+        │   └── tls-localhost-7053.pem
+        └── user
+  </pre>
+
+**Impact if expired**: Production outage. Ordering nodes are no longer allowed to participate in cluster.
+
+[Reference](https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#enroll-orderer)
+
+
+### Admin Certificates
+
+Ordering Service Organization Channel Admin Certificates and Peer Service Organization Channel Admin Certificates are issued for each organization.
+
+#### Ordering Service Organization Channel Admin Certificate
+
+**Description**: Certificate for an organization administrator to manage ordering service and channel updates.
+
+**Location**: Dependent on implementation:
+
+<pre>
+ordererca/
+└── ordereradmin
+└── msp
+    ├── admincerts
+    │   └── cert.pem
+    ├── cacerts
+    │   └── localhost-7053.pem
+    ├── keystore
+    │   └── key.pem
+    ├── signcerts
+    │   └── cert.pem
+    └── user
+</pre>
+
+**Impact if expired**: Transactions can continue to work successfully. Cannot modify channels from a client application or manage the orderer from the console.
+
+[Reference](https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#enroll-org0-s-admin)
+
+
+#### Peer Service Organization Channel Admin Certificate
+
+**Description** - Certificate for an organization administrator to manage a peer, including channel and chaincode services.
+
+**Location** - Dependent on implementation:
+
+<pre>
+org1ca/
+└── org1admin
+└── msp
+├── admincerts
+│     └── cert.pem
+├── cacerts
+│     └── localhost-7053.pem
+├── keystore
+│     └── key.pem
+├── signcerts
+│     └── cert.pem
+└── user
+</pre>
+
+**Impact if expired**: Transactions can continue to work successfully. Cannot install new smart contracts from a client application or manage the peer from the console.
+
+[Reference](https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#enroll-org1-s-admin)
+
+
+
+### Client Certificates
+
+**Description**: Two types of Client Certificates are issued for each organization:
+
+1. **Organization Enrollment Certificate** - Authenticates the client identity for communication with peers and orderers.
```

Comment:
I would reserve the word "communication" for the TLS Certificate. Maybe say "interactions" here instead.

```suggestion
1. **Organization Enrollment Certificate** - Authenticates the client identity for interactions with peers and orderers.
```

Reply:
ok.